src/data_access/FirebaseUserDataAccessObject.java

```diff
@@ -0,0 +1,38 @@
+package data_access;
```

Comment:
This file doesn't seem to be relevant to adding our basic chat message functionality, maybe pull it into a separate PR?

Reply:
Good point, opting to merge now instead so that other people can build on this